.github/workflows/tests-schedule.yml

```diff
@@ -0,0 +1,45 @@
+name: tests
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/tests-schedule.yml"
+      - "test/test_datasets_download.py"
+
+  schedule:
+    - cron: "0 4 * * *"
```

Comment:
With this the tests will be run at `04:00 UTC+0`. It would be useful to run this *after* the nightlies are uploaded to PyPI. @seemethere do you know at what time this is usually the case?

Reply:
Ours are typically kicked off at `7:00 UTC` and fully published at around `9:00 UTC`, just to be safe.